packages/shared/icons.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+
+// TODO: once we have an icon library, use that instead
+export const DownTriangle = () => (
+  <svg
```

Comment:
Can we think about how we could achieve this with svg sprites instead? I would like to save on the size of the dom tree.

Reply:
When we build out the icon distribution, sure :)

Reply:
So this is not here to stay great. :-)